analyzers/src/SonarAnalyzer.Common/Rules/PropertiesAccessCorrectFieldBase.cs

```diff
@@ -169,8 +181,11 @@ private IList<PropertyData> CollectPropertyData(IEnumerable<IPropertySymbol> pro
 
         private readonly struct PropertyData
         {
-            public PropertyData(IPropertySymbol propertySymbol, IEnumerable<FieldData> read, IEnumerable<FieldData> updated,
-                bool ignoreGetter, bool ignoreSetter)
+            public PropertyData(IPropertySymbol propertySymbol,
```

Comment:
NITPICK: This could have been a one liner as it does not go over the limit of 200 characters